tests/hypothesis/to_unit_test.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Did you try to see if you got any failures in these tests if you revert your changes?

Reply:
Yes, lots of errors.